Gemfile

```diff
@@ -33,11 +33,13 @@ gem "scenic"
 gem "scenic_sqlite_adapter"
 gem "scenic-mysql"
 
+gem 'activerecord-hierarchical_query', git: 'https://github.com/XertroV/activerecord-hierarchical_query', branch: 'rails-6-1'
```

Comment:
note: the code to get all ancestors/descendants is like 4 lines with ARHQ but like 20 lines with arel. should check if we can get arel out of arhq and evaluate long term use or not